examples/src/DynamoDB.hs

```diff
@@ -65,7 +67,9 @@ insertItem region secure host port table item = do
   let dynamo = setEndpoint secure host port DynamoDB.defaultService
 
   lgr <- newLogger Debug stdout
-  env <- newEnv discover <&> set envLogger lgr . within region . configure dynamo
+  env <-
+    newEnv discover
+      <&> set (field @"envLogger") lgr . configure dynamo . within region
```

Comment:
```suggestion
    newEnv discover <&> set #envLogger lgr . configure dynamo . within region
```
Seems this is a bit inconsistent intros and the following files.

Reply:
I'm specifically doing some of them with `OverloadedLabels` and some with raw `generic-lens` just so people can see both ways.